### ISSUE TITLE
Redirect after submit

### ISSUE THREAD

**Issue by pieter-groeneweg** [nextcloud/forms#548]
- Nextcloud-Version: 19.0.1
- Forms-Version: 2.0.2

I like to see this app evolving. My future intent is to use the app to create entry forms for sports events.
What I am missing after form submit is the possibility to return (redirect) to a(ny) page.

Current:

- A link to the (anonymous) form will be published on a site.
- Form is filled out and submitted.
- Thank you shows.
- Nothing happens.

Feature:
- A link to the (anonymous) form will be published on a site.
- Form is filled out and submitted.
- Thank you shows.
- **After x seconds it opens the predefined page on the site (or any site)**

**Comment by skjnldsv** [comment]
Easily doable I guess!

**Comment by skjnldsv** [comment]
@jancborchardt what would the ux be like?
An input ont he right sidebar?

**Comment by jancborchardt** [comment]
Some questions:

- Do Google Forms, Microsoft Forms, etc have this kind of functionality?
- What exactly is the use-case here, which kind of page would you like to redirect to here?
- Is this related to "Ability to embed form into website #324" – that is, if the form is already embedded into the site you would like to show afterwards, would this be a solution as well?

**Comment by pieter-groeneweg** [comment]
@jancborchardt 

- No, Google forms doesn't have this, M$ I wouldn't know. But even if they had it, I would not want to use any of these... Using these for applications and entries I need to check all their EULA's before publishing a form. One of the things we are proud of in our org is that we do not share any personal data with anyone but for what we stated in OUR pricacy declaration(s) and what we agreed on with the users and people involved. Let alone we have too litle resources to legally check all these 3rd party EULA's. We do allow some GA and FB cookies, but we don't want to go any beyond that for what they are allowed through our site.

- Our use case would be membership application form and entry forms for sports events (participants <18, needing parents consents)
A membership application should redirect to a "thank you for becoming a member" page.
An event X entry should redirect to an "event X info page". Several event forms can be available tp the public. Events may be open for non-members.

- If a form can be embedded like, for instance, a facebook timeline, if that is what is meant in #324, it would be awesome. But then it still needs some handler to catch the submit event to redirect to another page. Which can be a thank you page or the entry list of a particular event. (see https://github.com/nextcloud/forms/issues/553)

I hope any of this makes some sence.

Some additional side info.
We are working on a site for a youth sailing class.
We already have the calendar embedded from Nextcloud.
Race results are dropped in a nextcloud folder and automatically read and reformatted by a sites plugin. Then published
As we expect future racing events and organisations to change operation due to Covid we already started to become future proof. Participation management of events may be come more a task of the several Sailing Classes.

This is all very specific of course, but I can imagine more and more organisations may need forms and tools to adapt to the post covid era. Restaurants in NL need to register who is dining and when. Hairdressers need booking systems and on and on... 
I even can see use for the forms tool connected to the calendar app and contacts app.
